File: 5 lang progs/prog52a.jl


print("enter length ")
len = readline()
n1 = parse(Int64,len)

print("enter width ")
wid = readline()
n2 = parse(Int,wid)

area = n1 * n2 
perim = 2 *(n1 + n2)

println("The area is: ", area)
println("The perimiter is: ", perim)
/*
enter length 2
enter width 2
The area is: 4
The perimiter is: 8
*/

File: 5 lang progs/Prog52a.js
const prompt = require('prompt-sync')();

var len = Number(prompt(" what is the length"));
var wid = Number(prompt(" what is the width"));
var a = len * wid ;
var perim =  (2* len) + (2* wid);
console.log("the area is " + a);
console.log("the perim is " + perim);
/*
what is the length2
 what is the width2
the area is 4
the perim is 8
*/
